src/experiments.js

```diff
@@ -34,35 +34,15 @@ const COOKIE_MAX_AGE_DAYS = 180;  // 6 month
 /** @const {time} */
 const COOKIE_EXPIRATION_INTERVAL = COOKIE_MAX_AGE_DAYS * 24 * 60 * 60 * 1000;
 
-/** @const {string} */
-const CANARY_EXPERIMENT_ID = 'dev-channel';
-
 /** @type {Object<string, boolean>|undefined} */
 let toggles_;
 
 /**
- * Whether the scripts come from a dev channel.
- * @param {!Window} win
- * @return {boolean}
- */
-export function isDevChannel(win) {
-  if (isExperimentOn(win, CANARY_EXPERIMENT_ID)) {
```

Comment:
This PR is based on the proposal #6802 . I'm actually not 100% sure I'm doing the right thing, please take a look.